recipes/compute.rb

```diff
@@ -61,6 +61,22 @@
     not_if '/sbin/ppc64_cpu --smt 2>&1 | ' \
       'grep -E \'SMT is off|Machine is not SMT capable\''
   end
+when 'x86_64'
+  if node.deep_fetch('dmi', 'processor', 'manufacturer') == 'Intel(R) Corporation'
```

Comment:
You should be using ``only_if { node.deep_fetch('dmi', 'processor', 'manufacturer') == 'Intel(R) Corporation' }`` instead of wrapping it like this.